determined-ai/determined
# feat: Display Log retention days and Remaining log retention days in Logs Tab

## Ticket

DET-10242


## Description

In the Trials Overview page added a "Log Retention Days" field. 
In the logs tab, added a small tag that shows the number of days remaining to retain the logs


## Test Plan

1. Create 2 experiments, one with only one trial and another with multiple trials with different log retention days. Ex. `det e create const.yaml .  --config "retention_policy.log_retention_days=100" `
2. Go to the Single Trial Experiement Details Page and verify that the Log Retention Days and the Remaining log retention days on the Log tab, and its tool tip are accurate. 
<img width="1488" alt="Screen Shot 2024-05-21 at 12 50 55 PM" src="https://github.com/determined-ai/determined/assets/131198887/63c9d314-2a87-4ef5-b85a-83ecfad54b00">

3. Click on the 3 dots in the Right corner of the Single Trial Experiement Details Page and select Retain Logs, and change the log retention days of this Experiment and verify that the Log Retention Days and Remaining log retention days update correctly (it may take up to 5 seconds)
4. Go to the trial details page of each trial for the second experiment and make sure that the Log Retention Days and the Remaining log retention days on the Log tab are accurate.


## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
